### PR TITLE
Implement draft-ietf-opsawg-ipfix-on-path-telemetry

### PR DESCRIPTION
**Pull request by Yuya9786**
## Description

- Support for [draft-ietf-opsawg-ipfix-on-path-telemetry ](https://datatracker.ietf.org/doc/draft-ietf-opsawg-ipfix-on-path-telemetry/)
  - Obtain and export the following metrics
    - PathDelayMeanDeltaMicroseconds
    - PathDelayMaxDeltaMicroseconds
    - PathDelayMinDeltaMicroseconds
    - PathDelaySumDeltaMicroseconds
- Change the type of BPF MAP from HASH to PERF_EVENT_ARRAY so that the calculation of statistics moves out of xdp data plane
- Refactor the structures of the directory
  - pkg/packet/ipfix -> pkg/ipfix
  - pkg/bpf/srv6.go -> internal/pkg/meter/srv6.go

## Type of change

* [x] New features
* [ ] Bug fixes 
* [x] Refactoring
* [ ] Documentation updates

## Motivation and Context

To obtain and export on-path delay metrics

## How is This Tested?
 

Tested with packets with IOAM timestamps inserted by VPP (https://github.com/network-analytics/vpp/tree/feature/ipfix-onpath-telemetry)

## Other Information

Change the licence of xdp code to `Dual GPL/MIT` because `bpf_perf_event_output()` is a GPL-restricted function.

### CODE REVIEW

**Review by watal — APPROVED** [review]
Looks good, thanks!!

**Review by Motok1 — APPROVED** [review]
Thanks! 👍

File: cmd/fluvia/main.go [hunk 1]
```diff
@@ -51,5 +51,10 @@ func main() {
 		ingressIfName = c.Ipfix.IngressInterface
 	}
 
-	client.New(ingressIfName, raddr)
+	interval := c.Ipfix.Interval
```
Comment (Motok1) [inline cmd/fluvia/main.go:54]: It would be nice if the readme could understand how to use this option.

Comment (Yuya9786) [reply cmd/fluvia/main.go:54]: I wrote how to configure interval in getting-started.md

**Pull request merged**